src/components/metadata-modal/metadata-modal.js

```diff
@@ -89,6 +94,16 @@ const MetadataModal = ({ metadata, onToggle, visible }) => {
           </button>
         </div>
       )}
+      {hasJSON && (
```

Comment:
can we move this line 97 above line 82 and move the div from 83-95 within hasPlot (59-65) ?

Reply:
What effect will this have on the UI?

Reply:
no effect as such just a better structure to the code.